src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -282,7 +282,19 @@ ucs_status_t uct_rdmacm_cm_ep_disconnect(uct_ep_h ep, unsigned flags)
     uct_rdmacm_cm_ep_t *cep = ucs_derived_of(ep, uct_rdmacm_cm_ep_t);
     char ip_port_str[UCS_SOCKADDR_STRING_LEN];
 
-    ucs_trace("rdma_disconnect on cm_id %p", cep->id);
+    ucs_trace("%s: rdma_disconnect on ep %p cm_id %p",
+              (cep->flags & UCT_RDMACM_CM_EP_ON_SERVER ? "server" : "client"),
+              cep, cep->id);
+
+    if (ucs_unlikely(cep->flags & UCT_RDMACM_CM_EP_DISCONNECT_CALL)) {
```

Comment:
rename flag "UCT_RDMACM_CM_EP_DISCONNECT_CALL" to "UCT_RDMACM_CM_EP_DISCONNECTING"